instrumentation/grizzly-2/src/main/java/io/opentelemetry/auto/instrumentation/grizzly/GrizzlyHttpHandlerInstrumentation.java

```diff
@@ -66,39 +64,44 @@ public boolean defaultEnabled() {
   public static class HandleAdvice {
 
     @Advice.OnMethodEnter(suppress = Throwable.class)
-    public static AgentScope methodEnter(@Advice.Argument(0) final Request request) {
+    public static SpanScopePair methodEnter(@Advice.Argument(0) final Request request) {
```

Comment:
this is my first PR review in here, and I know this isn't totally the right place to mention this, but what about calling this thing a `SpanWithScope`, rather than explicitly calling out the pair nature?

Reply:
👍 `SpanWithScope` sounds good to me, I'll open an issue to track that